.github/workflows/CI.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           - { name: "gio", features: "v2_74", nightly: "--all-features", test_sys: false }
           - { name: "glib", features: "v2_74", nightly: "--all-features", test_sys: false }
           - { name: "graphene", features: "", nightly: "", test_sys: true }
-          - { name: "pango", features: "v1_50", nightly: "--all-features", test_sys: false }
```

Comment:
```
  cargo:warning=`"pkg-config" "--libs" "--cflags" "pango" "pango >= 1.52"` did not exit successfully: exit status: 1
  error: could not find system library 'pango' required by the 'pango-sys' crate

  --- stderr
  Package dependency requirement 'pango >= 1.52' could not be satisfied.
  Package 'pango' has version '1.50.13', required version is '>= 1.52'
```

Reply:
At some point pango upstream should probably get a bump to 1.51